<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.003, y1:0.017, x2:1, y2:0.983, stop:0 rgba(119, 203, 239, 255), stop:1 rgba(236, 211, 69, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 35pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>NUMBER LINK</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="NewGameButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
  background: #FFFFFF;
  padding: 20px;
  width: 200px;
  height: 150px;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49);
</string>
    </property>
    <property name="text">
     <string>НОВАЯ ИГРА</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ContinueButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>460</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
  background: #FFFFFF;
  padding: 20px;
  width: 200px;
  height: 150px;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)
</string>
    </property>
    <property name="text">
     <string>ПРОДОЛЖИТЬ</string>
    </property>
   </widget>
      <widget class="QPushButton" name="TopButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>560</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
  background: #FFFFFF;
  padding: 20px;
  width: 200px;
  height: 150px;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)
</string>
    </property>
    <property name="text">
     <string>ТОП ИГРОКОВ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>660</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 25px;
  background: #FFFFFF;
  padding: 20px;
  width: 200px;
  height: 150px;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color:rgb(67, 65, 49)
</string>
    </property>
    <property name="text">
     <string>ВЫХОД</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
